packages/block-editor/src/components/block-list/style.scss

```diff
@@ -85,6 +85,21 @@
 			opacity: 1;
 		}
 	}
+
+	// Active template part highlighting.
+	&.template-part-highlighting:not(.is-focus-mode) {
+		opacity: 0.5;
+		transition: opacity 0.1s linear;
+		@include reduce-motion("transition");
+
+		&.is-selected,
```

Comment:
Can we remove some of these classnames since we can assume that `is-active-template-part` means that it's selected?

Reply:
Ah yeah we probably don't need that one anymore 😁

Reply:
Looking again, I don't think we can remove any of these selectors and still have it functioning properly.

We could replace the `&.is-selected` with `&.is-active-template-part` if that seems more ideal to you?

Reply:
I personally think that makes sense, since only the active template part would receive opacity: 1, but it's up to you. Since it's functionally important we don't have to change it

Reply:
updated